controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Tasks.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pinterest.rocksplicator.controller.resource;
+
+import com.pinterest.rocksplicator.controller.bean.TaskBean;
+
+import java.util.List;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * @author Ang Xu (angxu@pinterest.com)
+ */
+@Path("/v1/tasks")
+public class Tasks {
+
+  /**
+   * Retrieves a task by the task id.
+   *
+   * @param id  task id
+   * @return    {@link TaskBean} or null if task doesn't exist
+   */
+  @GET
+  @Path("/{id : [a-zA-Z0-9\\-_]+}")
```

Comment:
task id will always be a number. because it will be genereated automatically by mysql auto increament